@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,23 @@
+declare interface PMGroupUpdateRequest {
+    prizeGroupName?: string;
+    prizeGroupDescription?: string;
+    prizeGroupOrder?: Array<string>;
+  }
+
+  declare interface PMGroupCreateRequest extends PMGroupUpdateRequest {
+    prizeGroupId: string;
+    prizeGroupName: string;
+    prizeGroupDescription: string;
+    prizeGroupOrder: Array<string>;
+  }
+
+  declare interface PMGroup extends PMGroupCreateRequest {
+    prizeGroupId: string;
+    prizeGroupName: string;
+    prizeGroupDescription: string;
+    prizeGroupOrder: Array<string>;
```

Comment:
Minor nitpick, but generally unless there is a really complicated type, it is generally easier to use `[]` to denote an array.
```suggestion
    prizeGroupOrder: string[];
```